src/providers/TransactionBuilderContextProvider.tsx

```diff
@@ -644,7 +650,117 @@ const TransactionBuilderContextProvider = ({
       setTransactionBlocks([]);
       showAlertModal('Transaction sent');
       setIsSubmitting(false);
-    } else {
+
+    }
+    else if (crossChainActions[0].type == TRANSACTION_BLOCK_TYPE.PLR_DAO_STAKE) {
+      let crossChainAction = crossChainActions[0];
+
+      if (!crossChainAction.receiveAmount) {
+        showAlertModal('Failed to get receiveAmount');
+        setIsSubmitting(false);
+        return;
+      }
+
+      let result: {
+        transactionHash?: string;
+        errorMessage?: string;
+      };
+
+      result = crossChainAction.useWeb3Provider
```

Comment:
please reuse what other blocks use besides `KLIMA_STAKE`, it is wrong example to follow